LunrCore/Index.cs

```diff
@@ -9,6 +9,8 @@
 using System.Text.Json.Serialization;
 using System.Threading;
 using System.Threading.Tasks;
+using Lunr.Globalization.fr;
+using Lunr.Globalization.it;
```

Comment:
So this is a little weird that this file would only get some additional `using`s. Is it due to some code moving, or is it some leftover you missed?

Reply:
These were just detritus I forgot to remove when I moved the builder methods.